src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -12,33 +12,44 @@
 namespace Symfony\Component\Form\Extension\Core\Type;
 
 use Symfony\Component\Form\AbstractType;
-use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
 use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
+use Symfony\Component\Form\ChoiceList\Loader\IntlCallbackChoiceLoader;
 use Symfony\Component\Intl\Intl;
+use Symfony\Component\OptionsResolver\Options;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class CountryType extends AbstractType implements ChoiceLoaderInterface
 {
     /**
-     * Country loaded choice list.
+     * BC layer.
```

Comment:
By marking it `@deprecated` instead, even if it's private, would help a bit when reading these classes from an IDE as the property usages in this class will be striked out.